addon/templates/components/polaris-range-slider.hbs

```diff
@@ -1,20 +1,20 @@
-{{#polaris-labelled
-  dataTestLabelled=dataTestRangeSlider
-  id=id
-  label=label
-  error=error
-  action=labelAction
-  labelHidden=labelHidden
-  helpText=helpText
-}}
+<PolarisLabelled
+  @dataTestLabelled={{this.dataTestRangeSlider}}
+  @id={{this.id}}
```

Comment:
@vladucu @andrewpye Is there a preferred way we want to fall back onto defaults for public attributes like this?

Right now I'm just using `this.` instead of `@`, but was thinking we might want to to do something like:

```
@id={{or @id this.id}}
```

for now during this transitional period just to be super explicit that we're falling back on a default? Same goes for `min`, `max`, `onFocus`, and `onBlur`.

Reply:
👍 
Love that...nice idea